save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/CreateOrganizationView.kt

```diff
@@ -89,7 +93,9 @@ class CreateOrganizationView : AbstractView<Props, OrganizationSaveViewState>(tr
                     post("$apiUrl/organization/save", headers, Json.encodeToString(newOrganizationRequest))
 
             if (responseFromCreationOrganization.ok) {
-                window.location.href = "${window.location.origin}#/${organizationName.replace(" ", "%20")}/"
```

Comment:
What was wrong with this redirect? Incorrect value of organization name?

Reply:
The name was fine. First problem is that we used `ButtonType.submit`. As far as I understood, redirects are not possible with it. Second issue, I assume, is that I changed `window.location.origin` inside of coroutine. When I moved it out of coroutine it started to work fine.

Reply:
Sounds a bit strange, but I can believe that some stuff can't be done from a coroutine.